src/dom.js

```diff
@@ -284,7 +284,7 @@ class dom {
 		var limit = event.currentTarget.parentNode;
 		var defFns = [];
 
-		while (currElement && currElement !== limit && !event.stopped) {
+		while (currElement && currElement !== limit && !event.stopped && !currElement.disabled) {
```

Comment:
Hi @mairatma. Thank you for your documentation. It's nice to, in case of doubts, to come back here and confirm what we want to achieve.

I began to investigate and I have some impressions of that.

(1) Yes you are right. The Disabled attribute only match with forms elements like: **buttons**, **inputs**, **selects**, **fieldsets** and so on. Browsers **ignore** the disabled attribute in elements like **divs**, **headers** and so on.

(2) Chrome ignore any mouse event on disabled elements but Firefox doesn't. Firefox just ignore **clicks**. The WC3 DOM documentation says nothing about mouse events besides **clicks**.

If you have some links to share, please, put here. 😃 

### Links to follow:
https://w3c.github.io/html/disabled-elements.html#disabled-elements